autoscale-workload-rabbit/src/main/java/com/hpe/caf/autoscale/workload/rabbit/RabbitWorkloadAnalyser.java

```diff
@@ -125,4 +144,18 @@ private ScalingAction getScalingAction(final ScalingOperation op, final int amou
             return ScalingAction.NO_ACTION;
         }
     }
+
+    @Override
+    public String retrieveEmailContent(final String percentageMem)
+    {
+        final String emailContenet
+            = "To whom it may concern, \n"
+            + "The RabbitMQ instance running on system " + System.getenv("CAF_RABBITMQ_MGMT_URL") + " is experiencing issues.\n"
+            + "\n"
+            + "RabbitMQ has used " + percentageMem + "% of its high watermark memory allowance.\n" 
+            + "\n"
+            + "From: Autoscaler\n";
```

Comment:
Is this "From: Autoscaler" line intentional?  I found it a bit weird at the bottom of the e-mail content.